fbmsgbot/models/template.py

```diff
@@ -15,6 +27,20 @@ def __init__(self, type, **kwargs):
         self.type = type
         self.kwargs = kwargs
 
+        if self.type == self.receipt_type:
+            assert 'recipient_name' in self.kwargs, \
```

Comment:
Could this not be a for loop aswell? 

for prop in prop
assert prop in kwargs?